app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
```

Comment:
What is a `vehicle_id`? Is it like `vehicle_id` of 1 is SEPTA, 2 is something else, etc? If so, I think we should create a `VehicleType` model and have a relationship between  `IdlingIncident` and `VehicleType`. `IdlingIncident` can have an integer id (this can be the primary key) as well as a string name.